### DISCUSSION TITLE
Pacage names are displayed incorrectly

### DISCUSSION THREAD

**Discussion by zdm** [community/community#62352] [Packages] [answered]
### Select Topic Area

Bug

### Body

Hi.

I have organization - zerocluster;

In this oprgnozation I have repositories with the following packages:
```
- zerocluster/node;
    - ghcr.io/zerocluster/node;
    - ghcr.io/zerocluster/node/app;

- zerocluster/postgresql;
    - - ghcr.io/zerocluster/postgreql/15;
```

Previusly this packages were displayed in the organization packages list as:
```
- node
- node/app
- postgresql/15
```

But strting from a few days ago the started displaying as:
```
- 15
- node
- app
```

This is confusiong.

Could you please help me to understans os this is nome my mistake or you made changes on your side?

Thank you.

**Comment by vatche-t** [comment, marked as answer]
Based on the information you provided, it seems that the packages in your organization, "zerocluster," are displayed differently than they were before. Previously, they were shown as:

node

node/app

postgresql/15

But now, they appear as:

15

node

app

This alteration has the potential to originate from diverse justifications. Here are some possible explanations:

Package Manager Updates: The display of packages might be controlled by the package manager you are using (e.g., npm, Docker Hub, etc.). Sometimes, package managers update their interface or sorting algorithms, which could result in changes to the display of package names.

Any alteration in the repository titles or adjustment to their metadata descriptions could bring about a distinct presentation of the package names.

Versioning Changes: If versioning has been updated for the packages, it might affect how they are displayed in the package list.

To identify the specific reason for the changed display, you can start by checking the documentation or release notes of the package manager you are using. It might provide information about any recent updates that could have affected the display.

If you did not make any adjustments to your repositories or packaged arrangements, it is probable an outer element like an adjustment to the packaged director. Yet, you might necessity to check further to pinpoint the actual reason.

**Comment by vatche-t** [comment]
also, I will be happy that if you could Kindley accept the answer.

**Comment by zdm** [comment]
Hmm, this packages are plain docker images.
I use docker to build and push them.
Docker wasn;t updated last months.

It takes package name from the docker image name, there are no other options.
Images names not changed.

So, I don't understand what options I can modify to make it works correctly.

**Comment by zdm** [comment]
Also, thank you very much for your answer.

**Comment by vatche-t** [comment]
Since you are dealing with Docker images and the image names have not changed, there might be another possible explanation for the changed display in your package list.

One possibility is that the registry or platform where you host your Docker images might have undergone some changes in how they display image names in the repository. Docker Hub, GitHub Container Registry (GHCR), or other registry platforms could update their interfaces, which could affect how the package names are shown in the package list.

**Comment by zdm** [comment]
The problem, that I really dont see any options, which I can change.
Image has name and labels only.
Labels are standrd in my case and dorsm't contains anything, that can be related to the package name.

```
"Labels": {
    "git.branch": "main",
    "git.date": "2023-07-31T19:55:49+03:00",
    "git.hash": "907057e45f8b92e1f339ee6bfa5c1850778721c7",
    "git.tags": "latest,next,v2.18.7",
    "org.opencontainers.image.description": "branch: main, tags: [latest,next,v2.18.7], date: 2023-07-31T19:55:49+03:00, hash: 907057e45f8b92e1f339ee6bfa5c1850778721c7",
    "org.opencontainers.image.ref.name": "ubuntu",
    "org.opencontainers.image.source": "https://github.com/zerocluster/postgresql",
    "org.opencontainers.image.version": "22.04"
}
```

Only `"org.opencontainers.image.source": "https://github.com/zerocluster/postgresql",` can be changed.
But in my case it set correctly, according to the documentation ot should point to the image source.
And I didn;t changed this label for a long time.

Ok, I will try to play woth this label.

Thank you.

I don't know, how this community works. If you will get some bonuses - I can accept your answer,

**Reply by vatche-t** [reply]
you are welcome

**Comment by vatche-t** [comment]
for this community all you can do is accept answer thank you very much
